lib/service_check.pm

```diff
@@ -46,10 +46,12 @@ our @EXPORT = qw(
 );
 
 our $hdd_base_version;
-our $support_ver_def  = '11+';
+our $support_ver_def  = '12+';
 our $support_ver_12   = '=12';
 our $support_ver_ge15 = '15+';
 our $support_ver_ge12 = '12+';
+our $support_ver_ge11 = '11+';
+our $support_ver_le12 = '<=12';
```

Comment:
Have you met a service need several segments for the support_ver? If so, we need enhance here to support such scenario.

Reply:
no need till now, for old version of sles, we just have 11sp4 to 12 segment.